sw/device/silicon_creator/rom/BUILD

```diff
@@ -348,23 +356,29 @@ cc_test(
     ],
 )
 
-exclude_files(
-    name = "pre_package",
-    testonly = True,
-    srcs = [":rom_with_fake_keys"],
+[exclude_files(
+    name = "pre_package_{}".format(authenticity),
+    testonly = testonly,
+    srcs = [":rom_with_{}_keys".format(authenticity)],
     exclude_suffix = [
         ".ll",
         ".i",
         ".s",
     ],
-)
+) for authenticity, testonly in (
+    ("fake", True),
+    ("real", True),
```

Comment:
After we generate the real keys, this should change to `False`, correct? (and in the `pkg_files` rules below)

Reply:
Correct. Resolving now since it would be hard to miss when we start enforcing `testonly = False` in queries/tests/etc